tests/e2e/v2store_deprecation_test.go

```diff
@@ -182,6 +184,74 @@ func TestV2DeprecationSnapshotRecover(t *testing.T) {
 	assert.NoError(t, epc.Close())
 }
 
+func TestV2DeprecationSnapshotRecoverOldVersion(t *testing.T) {
```

Comment:
This looks like a standard downgrade test. Have you seen the test in https://github.com/etcd-io/etcd/blob/a708bed336bb95446f1a00e3cf1e0e8858af0cb4/tests/e2e/cluster_downgrade_test.go#L22 ? Why do we need a separate test for V2 backward compatibility?

Reply:
https://github.com/etcd-io/etcd/blob/a708bed336bb95446f1a00e3cf1e0e8858af0cb4/tests/e2e/cluster_downgrade_test.go#L22 test does not seem to create a snapshot and kvs. 
In next changes when we change how the .snap is published by 3.6, we need a test that recovers from .snap and verifies membership match.

Reply:
please let me know if I should add the test to cluster_downgrade_test.go instead of this file.